blocks/query-panel/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CategorySelect from './category-select';
+import RangeControl from '../inspector-controls/range-control';
+import SelectControl from '../inspector-controls/select-control';
+
+const DEFAULT_MIN_ITEMS = 1;
+const DEFAULT_MAX_ITEMS = 100;
+
+export default function QueryPanel( {
+	category,
+	numberOfItems,
+	order,
+	orderBy,
+	maxItems = DEFAULT_MAX_ITEMS,
+	minItems = DEFAULT_MIN_ITEMS,
+	onCategoryChange,
+	onNumberOfItemsChange,
+	onOrderChange,
+	onOrderByChange,
+} ) {
+	return (
+		<div>
```

Comment:
If the `div` is not serving a purpose on its own, wondering if we ought to just return as an array, with the only gotcha being we'd have to add keys to each rendered element, but this would reduce the total number of DOM nodes on the screen and reduce indentation in the component:

```js
return [
	<SelectControl
		key="..."
	// ...
];
```